### PR TITLE
Disabled the export button #317

### PR DESCRIPTION
**Pull request by MarcosApodaca**
solved the problem of disabling the export button.

#165
#307
#317

### CODE REVIEW

File: src/pods/toolbar/components/export-button/export-button.component.tsx [hunk 1]
```diff
@@ -125,6 +125,7 @@ export const ExportButton = () => {
       label="Export"
       onClick={handleExportClick}
       className={classes.button}
+      disabled={!canvasSchema.selectedElementId}
```
Comment (brauliodiez) [inline src/pods/toolbar/components/export-button/export-button.component.tsx:128]: I think this won't work, just load the table, do not select any table you will get this failing.

I guess it should be checking if there is at least one table

Check with @deletidev

Comment (JTrillo) [reply src/pods/toolbar/components/export-button/export-button.component.tsx:128]: Exactly, Braulio is right. Please @MarcosApodaca check this file: https://github.com/Lemoncode/mongo-modeler/blob/vnext/src/pods/toolbar/components/save-button/save-button.component.tsx. Desired behaviour is quite similar. If you have any questions, send me a message in slack 😉

**Comment by JTrillo** [comment]
Apart from comments above, you are not solving #317 in this PR. Did you forget to add some commits in the PR?

**Review by JTrillo — APPROVED** [review]

**Comment by JTrillo** [comment]
Good job!

**Pull request merged**